subdomains.md

```diff
@@ -27,6 +27,15 @@ In the website repository:
 1. Add your domain to [this list](https://github.com/usegalaxy-eu/infrastructure/blob/main/dns.tf#L24) under `subdomain`, and increase the `count` parameter below by one.
 2. Make a PR with these changes.
 
+### To add GxIT privileges
+
+To allow or grant your domain the GxIT privileges, the following needs to be done,
+
+1. Create a PR, adding your domains IT wildcard to the [sn06.yml playbook](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/sn06.yml#L41) like done in [this PR](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/916)
```

Comment:
```suggestion
1. Create a PR, adding your domains IT wildcard to the [sn06.yml playbook](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/5ecffc1153fefc8f4ff1e44855ebf66a73c0593d/sn06.yml#L34) like done in [this PR](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/916)
```

I suggest to use a permalink so when lines are added or removed the link still makes sense (feel free to use another line).